### ISSUE TITLE
Upgrade to Opal 0.9.0.beta2

### ISSUE THREAD

**Issue by Mogztter** [asciidoctor/asciidoctor.js#142]
All JavaScript engines (including Nashorn) seem to be running fine with Opal 0.9.0.beta2

**Comment by mojavelinux** [comment]
\o/

**Comment by Mogztter** [comment]
```
Running Nashorn jjs spec/share/basic.js... OK in 377.49384299999997 ms
Running Nashorn jjs spec/share/asciidoctor-convert.js...
Running Nashorn jjs spec/share/asciidoctor-convert.js... OK in 4280.277384 ms
Compiling spec/nashorn/BasicJavascriptWithNashorn.java into build...
Compiling spec/nashorn/BasicJavascriptWithNashorn.java into build... OK
Running Nashorn java BasicJavascriptWithNashorn...
Running Nashorn java BasicJavascriptWithNashorn... OK in 343.968669 ms
Compiling spec/nashorn/AsciidoctorConvertWithNashorn.java into build...
Compiling spec/nashorn/AsciidoctorConvertWithNashorn.java into build... OK
Running Nashorn java AsciidoctorConvertWithNashorn...
Running Nashorn java AsciidoctorConvertWithNashorn... OK in 3909.319585 ms
```

Victory !

**Comment by mojavelinux** [comment]
Joyous day! Nice work!

**Comment by Mogztter** [comment]
Resolved with #143 in 4cb32ed353fe3d2403c218c7c5885140d1262380
